test/test/components/previews/layouts/application.html.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Action View Component Test</title>
+  <style>
+    .preview-component {
```

Comment:
I'd be interested in having this test layout, if it is necessary, use the `stylesheet_tag` helper to load the main CSS assets in the demo app. I'm wary of having this example define custom styles for the components.

Reply:
Gotcha 👌